angular/projects/spark-angular/src/lib/components/sprk-card/sprk-card.stories.ts

```diff
@@ -37,6 +51,33 @@ const modules = {
 };
 
 export const defaultStory = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-card idString="default">
+      <sprk-stack sprkCardContent sprkStackItem itemSpacing="medium">
+        <p sprkStackItem sprkText variant="bodyTwo">
+          Default Card
+        </p>
+      </sprk-stack>
+    </sprk-card>
+  `,
+});
+
+defaultStory.story = {
+  name: 'Default',
+  parameters: {
+    docs: { iframeHeight: 150 },
+    jest: [
+      'sprk-card.component',
+      'sprk-card-content.directive',
+      'sprk-stack.component',
+      'sprk-stack-item.directive',
+      'sprk-text.directive',
+    ],
+  },
+};
+
+export const legacyStory = () => ({
```

Comment:
The convention we've been following with the legacy stories so far has been to list them all at the end, like we did with checkboxes
![Screen Shot 2021-03-04 at 9 43 47 AM](https://user-images.githubusercontent.com/15703773/109980652-20c7bc00-7cce-11eb-97a0-d51ea2dd4d5a.png)

Reply:
oh yeah!! I forgot about that. I'll do that. Thank you!